modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,124 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-creating-arm64-bootimage_{context}""]
+
+= Creating an `arm64` bootimage using the Azure image gallery
+ 
+To configure your heterogeneous cluster, you must create an `arm64` bootimage and add it to your Azure machine set. The following procedure describes how to manually generate an `arm64` bootimage. 
+ 
+.Prerequisites
+
+* Installed the Azure CLI (`az`).
+* Created a single-architecture Azure installer-provisioned cluster with the heterogeneous installer binary. 
+
+.Procedure
+. Login to your Azure account: 
++
+[source,terminal]
+----
+$ az login
+----
+. Create a storage account and upload the `arm64` virtual hard disk (VHD) to your storage account. The {product-title} installation program creates a resource group, however, the bootimage can be uploaded to a custom named resource group: 
++
+[source,terminal]
+----
+$ az storage account create -n <storage-account> -g <resource-group> -l westus --sku Standard_LRS <1>
+----
++
+<1> The `westus` object is an example region. 
++
+. Create a storage container using the storage account you generated:
++
+[source,terminal]
++
+----
+$ az storage container create -n <container> --account-name <storage-account>
+----
+. You must use the {product-title} installation program JSON file to extract the URL and `arch64` VHD name:
+.. Extract the `URL` field and set it to `RHCOS_vhd_origin_url` as the file name:
++
+[source,terminal]
+----
+$ RHCOS_vhd_origin_url=$(./openshift-install coreos print-stream-json | jq -r '.architectures.aarch64."rhel-coreos-extensions"."azure-disk".url')
```

Comment:
Also is there a way we can use `jsonpath` instead of `jq`, we are trying to avoid using `jq` commands in our docs.

Reply:
I don't think the output option (-o) is available for the installer, is it @Prashanth684 ?

Reply:
as of today there is not. @kelbrown20 what typically do other docs use?

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output
Near the bottom of the list shows the alternative:
For - `$ oc get clusterversion -o json|jq ".items[0].spec"`
Use insead- `$ oc get clusterversion -o jsonpath='{.items[0].spec}{"\n"}'

Reply:
It might be different for `oc` commands

Reply:
the openshift-install command doesn't support -o like Alessandro said. could we still use jq for this?

Reply:
Looking into. Thank you for checking!

Reply:
We can keep the `jq` for the TP, but will have to fix come GA for this feature